package_manifest.yaml

```diff
@@ -34,10 +34,10 @@ foreman_scl_packages:
   hosts:
     foreman:
       releasers:
-        - "{{ nightly_releaser }}"
-      build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
+        - '{{ nightly_releaser }}'
+      build_package_tito_releaser_args: '{{ nightly_package_tito_releaser_args }}'
       nightly_package_tito_releaser_args:
-        - "--arg jenkins_job=test_develop"
+        - --arg jenkins_job=test_develop
```

Comment:
I think this is the only change I don't like, but I can live with it if we can't teach the tool to do what I want.

Reply:
Redone with `preserve_quotes = True`